src/main/scala/dx/executor/wdl/WdlWorkflowSupport.scala

```diff
@@ -914,13 +924,23 @@ case class WdlWorkflowSupport(workflow: TAT.Workflow,
   ): BlockContext[WdlBlock] = {
     val block: WdlBlock =
       Block.getSubBlockAt(WdlBlock.createBlocks(workflow.body), jobMeta.blockPath)
-    // Some of the inputs could be optional. If they are missing,
-    // add in a None value.
-    val irInputEnv: Map[String, (Type, Value)] = block.inputs.collect {
-      case blockInput: WdlBlockInput if jobInputs.contains(blockInput.name) =>
-        blockInput.name -> jobInputs(blockInput.name)
-    }.toMap
-    val inputEnv = WdlUtils.fromIR(irInputEnv, wdlTypeAliases)
```

Comment:
The [fromIR()](https://github.com/dnanexus/dxCompiler/blob/2f2d26843c4a9a3d8f094fbf038a22d5ff512074/src/main/scala/dx/core/languages/wdl/WdlUtils.scala#L657) function can be removed

Reply:
APPS-247